Packs/CortexXpanse/Wizards/wizard-Xpanse_ASM.json

```diff
@@ -0,0 +1,126 @@
+{
+    "id": "Xpanse Use Case Adoption Wizard",
+    "version": -1,
+    "modified": "2022-04-10T11:55:54.250933+03:00",
+    "fromVersion": "6.9.0",
+    "name": "Xpanse Use Case Adoption Wizard",
+    "description": "Welcome to the Deployment Wizard!  These steps will guide you through configuring your content pack so you'll have a working use case when the wizard finishes. Make sure to follow the steps in order, you can always continue the wizard where you left off. Learn more about this pack’s deployment in the \"Cortex XSOAR Administrator Guide\" -> \"Set up Your Use Case with the Deployment Wizard\".",
+    "dependency_packs": [
```

Comment:
We will want to add the following packs as dependencies as well:

- [Azure Compute](https://github.com/demisto/content/tree/master/Packs/AzureCompute)
- [AWS-EC2](https://github.com/demisto/content/tree/master/Packs/AWS-EC2)
- [Google Cloud Compute](https://github.com/demisto/content/tree/master/Packs/GoogleCloudCompute)

Reply:
working on that now.

Reply:
Theres actually more than just azure compute, there is microsoft graph user and the api. im trying to get them all worked out in the pack. you will see an update in the packs_meta data too